test/modules/test_joomla_scanner.py

```diff
@@ -10,6 +11,30 @@ class JoomlaScannerTest(ArtemisModuleTestCase):
     # The reason for ignoring mypy error is https://github.com/CERT-Polska/karton/issues/201
     karton_class = JoomlaScanner  # type: ignore
 
+    def test_is_version_old(self) -> None:
```

Comment:
Don't we also have to mock time because `datetime.datetime.utcnow().replace(tzinfo=pytz.utc) - datetime.datetime.fromisoformat(release["published_at"])` may change over time?

Reply:
we didn't, but I added a test that actually check the date difference feature